src/Compilers/CSharp/Test/Emit3/Attributes/AttributeTests_StructLayout.cs

```diff
@@ -722,5 +769,310 @@ public void Bug1075326()
             VerifyStructLayout(@"struct S { int P { get; set; } }", hasInstanceFields: true);
             VerifyStructLayout(@"delegate void D(); struct S { event D D; }", hasInstanceFields: true);
         }
+
+        [Fact]
+        public void ExtendedLayoutAttribute_UnusedFields_NoDiagnostic()
+        {
+            string source = """
+                using System.Runtime.InteropServices;
+
+                [ExtendedLayout(ExtendedLayoutKind.CStruct)]
+                struct StructWithExtendedLayout
+                {
+                    public int i;
+                }
+                """;
+
+            var comp = CreateCompilation(
+                source,
+                references: [MinimalCoreLibBuilder.Create(ExtendedLayoutAttribute).ToMetadataReference()],
+                targetFramework: TargetFramework.Empty);
+            comp.VerifyDiagnostics();
```

Comment:
Please use `CompileAndVerify`, and:

* Pass a symbol validator that retrieves the symbol from the compiled module and verifies the layout on it from metadata. You may want to extract this into a helper for all the new tests.
* Call `VerifyDiagnostics` on it.

Reply:
This wasn't addressed, at least at the location that I left this comment.